src/wallet/feebumper.cpp

```diff
@@ -111,20 +111,22 @@ Result CreateTransaction(const CWallet* wallet, const uint256& txid, const CCoin
         return Result::WALLET_ERROR;
     }
 
-    // figure out which output was change
-    // if there was no change output or multiple change outputs, fail
-    int nOutput = -1;
-    for (size_t i = 0; i < wtx.tx->vout.size(); ++i) {
-        if (wallet->IsChange(wtx.tx->vout[i])) {
-            if (nOutput != -1) {
-                errors.push_back("Transaction has multiple change outputs");
-                return Result::WALLET_ERROR;
+    int nOutput = reduce_output;
+    if (nOutput == -1) {
+        // figure out which output was change
+        // if there was no change output or multiple change outputs, fail
+        for (size_t i = 0; i < wtx.tx->vout.size(); ++i) {
+            if (wallet->IsChange(wtx.tx->vout[i])) {
+                if (nOutput != -1) {
+                    errors.push_back("Transaction has multiple change outputs (use reduce_output=<index> to select the one to use)");
+                    return Result::WALLET_ERROR;
+                }
+                nOutput = i;
             }
-            nOutput = i;
         }
     }
     if (nOutput == -1) {
-        errors.push_back("Transaction does not have a change output");
+        errors.push_back("Transaction does not have a change output (use reduce_output=0 to use the single output)");
```

Comment:
This error will also display in the gui. Not sure if it makes sense there to mention reduce_output

Reply:
Oh. Hm..

Reply:
Do we have RPC's to change settings during runtime that are also config options?

Reply:
It also sounds confusing when there isn't a single output?

Reply:
I'll just revert it.